float8_experimental/float8_utils.py

```diff
@@ -28,12 +28,25 @@
 IS_AMD = torch.cuda.is_available() and torch.version.hip is not None
 
 
+# Helper functions to get individual F8 types based on backend architecture
```

Comment:
Would it be possible to put this into configuration instead of setting it dynamically?  It can be unexpected for numerics to change based on the environment.  It would also be good to support numerical emulation of all of these types regardless of whether the user's machine supports a float8 matmul.

Reply:
I'm afraid I don't understand your question. These helper functions are simply intended to grab the "right" version of the prebuilt torch F8 types. Could you elaborate on the change you'd like to see?

Reply:
Sure, it's just making the dtype flavors encoded in configuration instead of environment dependent.  Having this in configuration would make it easier to debug numerics without having the target hardware.

```
# float8 dtypes have a default which can be changed explicitly
config = ...
config.float8_flavors = 'nuz'
do_float8_things(..., config)
```

versus

```
# float8 dtypes magically change based on the environment
do_float8_things(...)
```

That said, my comment is not high pri, feel free to land and we can adjust this later if it becomes important.

Reply:
@vkuzo Sorry, got wrapped up in other work recently and just circled back to this. Okay, I will add an option in float8_experimental/float8_experimental/config.py and instead of checking the backend architecture, the code will check against this user-settable config variable